packages/engine-core/src/data-proxy/errors/DataProxyAPIError.ts

```diff
@@ -13,5 +13,12 @@ export abstract class DataProxyAPIError extends DataProxyError {
     super(message, info)
 
     this.response = info.response
+
+    // add request id to response message if it is present in the response header
+    const requestId = this.response.headers?.['Prisma-Request-Id']
+    if (requestId) {
+      const messageSuffix = `(The request id was: ${requestId})`
```

Comment:
Would it be better to have a property with the requestId, or do we want it to be always logged?

Reply:
We want it always to be logged so that users can easily provide those to support. But we can add this property additionally if you like to?

Reply:
That makes sense. Thanks, I think we're good then.